packages/mdc-checkbox/_mixins.scss

```diff
@@ -324,18 +322,77 @@ $mdc-checkbox-ripple-target: ".mdc-checkbox__ripple";
   }
 }
 
-@mixin mdc-checkbox-ink-color($color, $query: mdc-feature-all()) {
+///
+/// Customizes disabled state stroke and fill colors
+/// @param {Color} $unmarked-stroke-color - The desired stroke color for the unmarked state
+/// @param {Color} $unmarked-fill-color - The desired fill color for the unmarked state
+/// @param {Color} $marked-stroke-color - The desired stroke color for the marked state
+/// @param {Color} $marked-fill-color - The desired fill color for the marked state
+///
+@mixin mdc-checkbox-disabled-container-colors(
```

Comment:
As discussed, it's worth investigating to create disabled selector mixin.

See example here:
https://www.sassmeister.com/gist/9168132c52a31cd48917e64beb9e5a63

Reply:
Separated core styles from selector logic in both this change (checkbox) and radio, as later discussed.